src/com/tekion/cricket/Main.java

```diff
@@ -0,0 +1,28 @@
+package com.tekion.cricket;
+
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args)
+    {
+        int choice;
+        System.out.println("Press 1 to start a Match , Press 2 to start a series");
+        Scanner sc =  new Scanner(System.in);
+        choice = sc.nextInt();
+
+        switch(choice){
+            case 1:
+                Match match = new Match();
+                match.startMatch();
+                match.printScoreCard();
+                break;
+            case 2:
+                break;
```

Comment:
are we going to have any implementation in this `case 2` ?

Reply:
Yes sir, case 2 is for a series of matches. I have implemented the same. will push it in the week-2 branch.